src/Sdk/Cerbos/Sdk/CheckResult.cs

```diff
@@ -17,15 +18,28 @@ namespace Cerbos.Sdk
     public class CheckResult
     {
         private readonly Dictionary<string, Effect> _effects;
+        public CheckResourcesResponse.Types.ResultEntry.Types.Resource Resource { get; }
+        public CheckResourcesResponse.Types.ResultEntry.Types.Meta Meta { get; }
+        public List<OutputEntry> Outputs { get; }
```

Comment:
Some of the SDKs have convenience methods to get an output value by its key. Might be worth doing that in this SDK too.